docs/DeveloperGuide.md

```diff
@@ -427,6 +428,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: UC7 - Export contact list**
+
+**MSS**
+1.  Actor requests to export contact list to a specific file.
+2.  System exports the contact information into the file.
+
+    Use case ends.
+
+**Extensions**
+
+- 1a. System detects that the directory does not exist.
+
+    - 1a1. System creates the directory.
+
+      Use case resumes from step 2.
+
+- 2a. System detects that the file is used by another process.
+
+    - 2a1. System shows an error message.
+
+      Use case ends.
```

Comment:
Should this be 1a? Cause 2a means that the system would have exported the details already, before detecting the file being used.

Reply:
Alright I mistook extensions as it can happen during a step

I change it to 2b